sdc/datatypes/hpat_pandas_stringmethods_functions.py

```diff
@@ -1048,31 +1031,155 @@ def hpat_pandas_stringmethods_isdecimal_impl(self):
     return hpat_pandas_stringmethods_isdecimal_impl
 
 
+@sdc_overload_method(StringMethodsType, 'capitalize')
+def hpat_pandas_stringmethods_capitalize(self):
+    ty_checker = TypeChecker('Method capitalize().')
+    ty_checker.check(self, StringMethodsType)
+
+    def hpat_pandas_stringmethods_capitalize_impl(self):
+        item_count = len(self._data)
+        result = [''] * item_count
+        for idx in numba.prange(item_count):
+            result[idx] = self._data._data[idx].capitalize()
+
+        return pandas.Series(result, self._data._index, name=self._data._name)
+
+    return hpat_pandas_stringmethods_capitalize_impl
+
+
+@sdc_overload_method(StringMethodsType, 'title')
+def hpat_pandas_stringmethods_title(self):
+    ty_checker = TypeChecker('Method title().')
+    ty_checker.check(self, StringMethodsType)
+
+    def hpat_pandas_stringmethods_title_impl(self):
+        item_count = len(self._data)
+        result = [''] * item_count
+        for idx in numba.prange(item_count):
+            result[idx] = self._data._data[idx].title()
+
+        return pandas.Series(result, self._data._index, name=self._data._name)
+
+    return hpat_pandas_stringmethods_title_impl
+
+
+@sdc_overload_method(StringMethodsType, 'swapcase')
+def hpat_pandas_stringmethods_swapcase(self):
+    ty_checker = TypeChecker('Method swapcase().')
+    ty_checker.check(self, StringMethodsType)
+
+    def hpat_pandas_stringmethods_swapcase_impl(self):
+        item_count = len(self._data)
+        result = [''] * item_count
+        for idx in numba.prange(item_count):
+            result[idx] = self._data._data[idx].swapcase()
+
+        return pandas.Series(result, self._data._index, name=self._data._name)
+
+    return hpat_pandas_stringmethods_swapcase_impl
+
+
+@sdc_overload_method(StringMethodsType, 'casefold')
+def hpat_pandas_stringmethods_casefold(self):
+    ty_checker = TypeChecker('Method casefold().')
+    ty_checker.check(self, StringMethodsType)
+
+    def hpat_pandas_stringmethods_casefold_impl(self):
+        item_count = len(self._data)
+        result = [''] * item_count
+        for idx in numba.prange(item_count):
+            result[idx] = self._data._data[idx].casefold()
+
+        return pandas.Series(result, self._data._index, name=self._data._name)
+
+    return hpat_pandas_stringmethods_casefold_impl
+
+
+see_also_for_ismethods = '''
```

Comment:
Lets reduce the name 
```suggestion
seealso_ismethods = '''
```

Reply:
Also, please, follow the standard
For triple-quoted strings, always use double quote characters to be consistent with the docstring convention in P[EP 257 ](https://www.python.org/dev/peps/pep-0257/)